packages/portal-proto/src/features/cohortComparison/CohortCard.tsx

```diff
@@ -72,6 +72,20 @@ const CohortCard: React.FC<CohortCardProps> = ({
         </div>
       </div>
       <hr />
+      {!casesFetching && caseSetIds.length !== 0 ? (
+        <CohortVennDiagram caseSetIds={caseSetIds} cohorts={cohorts} />
+      ) : (
+        <VennDiagram
+          chartData={[
+            { key: "S1_minus_S2", value: 0, highlighted: false },
+            { key: "S2_minus_S1", value: 0, highlighted: false },
+            { key: "S1_intersect_S2", value: 0, highlighted: false },
+          ]}
+          labels={["S₁", "S₂"]}
+          ariaLabel="The Venn diagram displays the intersections, unions, or differences in the cohorts or sets. Additional information can be found in the Summary Table and the Overlap Table."
```

Comment:
label for cohort comparison should be:
`The Venn diagram displays the number of cases shared between the cohorts.`